.circleci/config.yml

```diff
@@ -78,13 +79,28 @@ jobs:
             npm i -g get-firefox
             get-firefox --branch nightly --platform linux --extract --target ~/
             ~/firefox/firefox -v
+            wget -O /tmp/geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v$GECKODRIVER/geckodriver-v$GECKODRIVER-linux64.tar.gz
+            mkdir $HOME/geckodriver && tar xvf /tmp/geckodriver.tar.gz -C $HOME/geckodriver
+            export PATH=$HOME/geckodriver:$PATH
+            firefox --version
+            geckodriver --version
       - run:
           name: Run npm test
           command: |
             . "$NVM_DIR/nvm.sh"
             nvm use 8.10.0
             export PATH=~/firefox:$PATH
             FIREFOX_CHANNEL=NIGHTLY MOZ_HEADLESS=1 npm test
+      - run:
+          name: Run Accessibility Tests
+          command: |
+            pip install -U pip
+            pip install tox
```

Comment:
These should be runnable locally, probably as `npm run test:accessibility` (`test:accessibility` is just a shell script listed in `package.json`)

It should use the same `make .venv` pattern that `test:server` uses. I suspect this will make the tox parts unnecessary. From there you just use things like `./.venv/bin/pytest ...` to access the specific tools you installed.

Reply:
Alrighty. I've read through the code, but I wasn't exactly sure how to implement the `.venv` for my tests--perhaps we could schedule a meeting to go over some of changes you've suggested.